exercises/concept/library-of-luton/.docs/hints.md

```diff
@@ -0,0 +1,25 @@
+# Hints
+
+## 1. Get first letter of title
+
+- There is [a method][index] that returns a Char from a String at a given index.
```

Comment:
```suggestion
- There is [a method][index] that returns a `Char` from a `String` at a given index.
```

This is also a general suggestion: anytime the docs mention something that can be used as actual code (like types), please put it in backticks. This will help students identify the actual language concepts.